client/src/classes/item.ts

```diff
@@ -12,15 +13,18 @@ export class Item {
   private unit: Unit;
 
   constructor(private name: string, public bought: boolean = false, 
-              public department: Department = null) {
+              public department: Department = null, private template: boolean = false) {
```

Comment:
What does template represent? Some existance of a global templating function for this Item?

It's should have no knowledge of what exists outside of them, since things outside might change and forget to tell the item.